src/Apps/Artist/Routes/Overview/Components/ArtistCareerHighlights.tsx

```diff
@@ -55,7 +59,7 @@ const ArtistCareerHighlights: FC<ArtistCareerHighlightsProps> = ({
         </Media>
       )}
 
-      {insights.length > 0 && (
+      {insights.length > 0 && hasEntitiesOrDescription && (
```

Comment:
I wanted to add the specs for this here, but whenever I try to mock the Relay store and artist, I consistently end up with mocked values for the fields. It would be great if this could be handled more effectively.